### PR TITLE
corrected warning span placement

### PR DESCRIPTION
**Pull request by dsnyder-fairway**
## unchanged
screen shot 2018-02-27 at 1 20 38 pm

## changed
screen shot 2018-02-27 at 1 20 48 pm

### CODE REVIEW

File: webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html [hunk 1]
```diff
@@ -631,8 +635,8 @@ <h3>{{'aggregate-reports.form.section.preview.heading' | translate}}</h3>
 
 </ng-template>
 
-<ng-template #warning let-text>
-  <span class="alert alert-warning small pt-xs pr-xs pb-xs pl-xs ml-sm">
+<ng-template #warning let-text let-style>
```
Comment (dsnyder-fairway) [inline webapp/src/main/webapp/src/app/aggregate-report/aggregate-report-form.component.html]: oops. Didn't mean to leave this

**Comment by wtritch** [comment]
Huh.  I thought we wanted it next to the title?

**Comment by dsnyder-fairway** [comment]
@wtritch it can lead to some issues
image

**Comment by esotrope** [comment]
i feel like there should be a css rule to make the whole thing break to a new line. we could use the flex children thing i think

**Comment by wtritch** [comment]
Can we keep it next to the title even when displaying estimated row counts?  I’d prefer to avoid making the form any longer >_<

**Comment by dsnyder-fairway** [comment]
@esotrope you are correct! change coming

**Comment by dsnyder-fairway** [comment]
screen shot 2018-02-27 at 1 59 29 pm
screen shot 2018-02-27 at 1 59 40 pm
screen shot 2018-02-27 at 1 59 52 pm

**Pull request merged**